src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,36 @@
+import { createSlice, nanoid } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    BOOK_ADDED: {
```

Comment:
- [ ] I can see you used the Action type but you didn't use it as we expected. Have a look at this article and try to fix your mistake. [Link...](https://github.com/erikras/ducks-modular-redux#:~:text=//%20Actions%0Aconst%20LOAD%20%20%20%3D%20%27my%2Dapp/widgets/LOAD%27%3B%0Aconst%20CREATE%20%3D%20%27my%2Dapp/widgets/CREATE%27%3B%0Aconst%20UPDATE%20%3D%20%27my%2Dapp/widgets/UPDATE%27%3B%0Aconst%20REMOVE%20%3D%20%27my%2Dapp/widgets/REMOVE%27%3B)

Reply:
Thanks for the Honest review @Rustamxon7  i really much appreciate it. Nevertheless, I am using Redux Toolkit with React Redux as instructed. (npm install react-redux @reduxjs/toolkit). The CreateSlice() Method carries the functionality of creating the reducer type and action type behind the scene.